main_routers/vrm_router.py

```
@@ -410,3 +412,214 @@ async def get_vrm_config():
+    "relaxed": ["relaxed", "joy", "fun", "content"],
+    "sad": ["sad", "sorrow", "grief"],
+    "angry": ["angry", "anger"],
+    "surprised": ["surprised", "surprise", "shock", "e", "o"]
+}
+
+
+def _get_emotion_config_path(model_name: str) -> Path | None:
+    """获取模型情感配置文件路径"""
+    import re
+
+    # 只允许安全的文件名字符：字母、数字、下划线、连字符
```

Ruff (0.15.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)